src/util/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+# =============================
+# Define targets
+# =============================
+add_executable(security-utilities_icvutil icvutil.c)
+add_executable(security-utilities::icvutil ALIAS security-utilities_icvutil)
+
+set_target_properties(security-utilities_icvutil PROPERTIES
+    EXPORT_NAME icvutil
+    OUTPUT_NAME icvutil
+)
+
+target_compile_features(security-utilities_icvutil PRIVATE c_std_90)
+set_target_properties(security-utilities_icvutil PROPERTIES C_EXTENSIONS ON)
+
+target_link_libraries(security-utilities_icvutil PRIVATE security-utilities::library secutils_compilation_options)
+
```

Comment:
Here is missing:
```
    OpenSSL::Crypto
    secutils_compilation_options
```

Apparently you did not try
```
SECUTILS_USE_ICV=1 cmake . && make
```

Reply:
I had to add: `$<$<BOOL:${SECURITY_UTILITIES_USE_UTA}>:security-utilities_uta-api>`